app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/ProfileViewModel.kt

```diff
@@ -11,7 +11,13 @@ import javax.inject.Inject
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.launch
+import ch.epfl.cs311.wanderwave.model.data.Track
 
+
+
+
+// Define a simple class for a song list
+data class SongList(val name: String, val tracks: MutableList<Track> = mutableListOf())
```

Comment:
Why do you need that ? Unfortunately I think it's gonna be hard to adapt the firebase connection if you use a specific dataclass for a list of tracks. Is it important ? This is a completely open quesiton if you think I should implement the songlist (which you should rename playlist btw), then I'll do it.

Reply:
Now that I think about it, the more general choice should be :

- The data class is better if we want "independant" playlists, that live outside of a beacon, that are like use playlists
- Keeping a normal List is better for backend storage and on device storage. and good if we keep playlists for a user or for a beacon.